src/ssl/ErrorDetailManager.cc

```diff
@@ -98,7 +98,7 @@ Ssl::ErrorDetailsList::Pointer
 Ssl::ErrorDetailsManager::getCachedDetails(const char * const lang) const
 {
     Cache::iterator it;
-    it = cache.find(lang);
+    it = cache.find(SBuf(lang));
```

Comment:
I hope you will follow up with a PR that will change `lang` type to SBuf by upgrading strHdrAcptLangGetItem() to return optional SBuf (instead of filling a fixed-size buffer).

Reply:
Eventually, yes. As a general approach, my strategy is:
1. remove SquidString, starting with low-hanging fruits
2. go chase char* and convert them as much as possible